api/config/cdr_config_test.json

```diff
@@ -143,6 +143,29 @@
       "microarrayVcfSingleSampleStoragePath": "microarray/vcf/single_sample",
       "accessTier": "controlled",
       "tanagraEnabled": true
+    },
+    {
+      "cdrVersionId": 9,
+      "isDefault": true,
+      "name": "Synthetic Dataset in the Controlled Tier v4",
```

Comment:
Does project.rb publish-cdr read this value? Can I change it after I  published (because right now it duplicates the value above it)?

Reply:
@freemabd @tarekmamdouh

Reply:
This needs to be v5. It's used in Datasetversion drop down when creating a workspace.
![Screen Shot 2024-11-04 at 4 32 05 PM](https://github.com/user-attachments/assets/c6b28388-417a-4916-b2b2-741d2465581e)

Reply:
@evrii And remember that dataset SC2023Q3R3 needs to be published before merging this PR

Reply:
Since this is the Test env, we can be somewhat arbitrary with the naming.  Could do "v8/Echo" or something.

But yes it does need to differ and be higher than the others.